cmd/hamctl/command/rollback.go

```diff
@@ -52,36 +56,10 @@ has no effect.`,
 					return fmt.Errorf("can't do rollback, because there isn't a release to rollback to")
 				}
 
-				funcMap := promptui.FuncMap
-				for name, f := range template.FuncMap() {
-					funcMap[name] = f
-				}
-				rollbackInteractiveTemplates.FuncMap = funcMap
-
-				items := mapToRollbackInteractiveTemplateData(releasesResponse.Releases)
-
-				searcher := func(input string, index int) bool {
-					release := items[index]
-					name := strings.ToLower(fmt.Sprintf("#%v %s", release.ReleaseIndex, release.ArtifactID))
-					input = strings.ToLower(input)
-
-					return strings.Contains(name, input)
-				}
-
-				prompt := promptui.Select{
-					Label:             fmt.Sprintf("Which release to rollback '%s' to?", environment),
-					Items:             items,
-					Templates:         &rollbackInteractiveTemplates,
-					Size:              10,
-					Searcher:          searcher,
-					StartInSearchMode: true,
-				}
-
-				index, _, err := prompt.Run()
-
+				index, err := selectReleaseUI(environment, releasesResponse.Releases)
 				if err != nil {
-					fmt.Printf("Rollback cancelled: %v\n", err)
-					os.Exit(1)
+					logger("something went wrong select release a release")
```

Comment:
This is a change in functionality, right? We would before just write "Rollback cancelled" and any error. If I hit CTRL+C in the prompt I wouldn't expect "Something went wrong"?

Reply:
Woops. I think it may've been a copy paste error, will fix